bash.html.markdown

```diff
@@ -36,7 +37,14 @@ VARIABLE="Some string"
 # But not like this:
 VARIABLE = "Some string"
 # Bash will decide that VARIABLE is a command it must execute and give an error
-# because it couldn't be found.
+# because it can't be found.
+
+# Or like this:
+Variable= 'Some string'
```

Comment:
Could you please all caps the 'Variable' name here to match the example? The PR you made with all caps can then discuss if we should keep it all caps or not but for this PR, lets do all caps please to keep the examples working. 

Thanks!